src/main.rs

```diff
@@ -680,6 +673,7 @@ impl<'c, 'probe> Registers<'c, 'probe> {
     }
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
This is something we should fix at some point, the code in probe-run has become pretty messy

```suggestion
#[allow(clippy::too_many_arguments)] // FIXME: clean this up
```

Reply:
Is this type of comment automatically recognized by some tool?

Reply:
Or should we maybe just open an issue for it, since this is more visible?